src/features/locators/locators.slice.ts

```diff
@@ -47,13 +50,34 @@ const locatorsSlice = createSlice({
       locatorsAdapter.addMany(state, payload);
     },
     changeLocatorAttributes(state, { payload }: PayloadAction<ChangeLocatorAttributesPayload>) {
```

Comment:
logic of this reducer becomes too complex. why you don't create a new reducer for this purpose?

Reply:
created